exercises/practice/accumulate/.meta/exemplar.erl

```diff
@@ -1,4 +1,4 @@
--module(example).
+-module(examplar).
```

Comment:
This has to be `example`. `exemplar` would be only for concept exercises (for which currently no infrastructure exists).

Please also rename the file accordingly.

Reply:
Done. 

BTW Is it possible to have more than one example file?

Reply:
Generally yes, though the current test script would only ever only handle `exemple`.

Allowing more would require changes in https://github.com/NobbZ/erl_exercism to recognize arbitrary example names, or the current test GHA script needs to get adjusted tp rename accordingly.